File: C#/uri1555.cs
using System;

class URI
{

    static void Main(string[] args)
    {
        int entrada = Convert.ToInt32(Console.ReadLine());

        for (int i = 0; i < entrada; i++)
        {
            string input2 = Console.ReadLine();
            string[] numero = input2.Split(' ');

            int x = Convert.ToInt32(numero[0]);
            int y = Convert.ToInt32(numero[1]);

            var rafael = Math.Pow((3 * x), 2) + Math.Pow(y, 2);
            var beto = 2 * Math.Pow(x, 2) + Math.Pow(5 * y, 2);
            var carlos = -100 * x + (Math.Pow(y, 3));

            if (rafael > carlos && rafael > beto)
            {
                Console.WriteLine("Rafael ganhou");
            }
            else if (carlos > rafael && carlos > beto)
            {
                Console.WriteLine("Carlos ganhou");
            }
            else
            {
                Console.WriteLine("Beto ganhou");
            }
        }


    }

}

File: python/uri1555.py



def rafael(x,y):
    return (3*x)**2 + y**2

def beto(x,y):
    return 2*(x)**2 + (5*y)**2

def carlos(x,y):
    return -100*x + y**3

entrada = input()

for i in range(0,int(entrada)):

 input2 = input()
 numero = input2.split(" ")

 x = 0
 y = 0




 try:
     y = int(numero[1])
     x = int(numero[0])
 except IndexError:
     pass

 if rafael(x,y) > beto(x,y) and rafael(x,y) > carlos(x,y):
    print("Rafael ganhou")
 if beto(x,y) > rafael(x,y) and beto(x,y) > carlos(x,y):
    print("Beto ganhou")
 if carlos(x,y) > beto(x,y) and carlos(x,y) > beto(x,y):
    print("Carlos ganhou")